Parse/cloud/main.js

```diff
@@ -15,6 +15,24 @@ Parse.Cloud.beforeSave(Parse.User, async (request) => {
   );
 });
 
+Parse.Cloud.afterSave(Parse.User, async (request) => {
+  const SearchItem = Parse.Object.extend("SearchItem");
+  const searchItem = new SearchItem();
+  var currentUser = request.object;
+  var username = await request.object.get("username");
+
+  searchItem.save({
+    searchName: username.toLowerCase(),
+    user: currentUser
+  })
+  .then((searchItem) => {
+    // The object was saved successfully.
+  }, (error) => {
+    // The save failed.
```

Comment:
We should do a `console.log` here (or I think in JS you can even do `console.error` right?)